src/chrome/content/rules/Yandex.net.xml

```diff
@@ -204,56 +176,39 @@ Fetch error: http://xss.yandex.net/ => https://xss.yandex.net/: (7, 'Failed to c
 		<!--
 			Miscellaneous:
 					-->
-		<exclusion pattern="^http://(?:direct|elements?|mobile\.photo|pythonbp)\.yandex\.net/" />
+		<exclusion pattern="^http://(?:api-yaru|direct|elements?|mobile\.photo|pythonbp)\.yandex\.net/" />
 
+			<test url="http://api-yaru.yandex.net/" />
 			<test url="http://direct.yandex.net/" />
 			<test url="http://element.yandex.net/" />
 			<test url="http://elements.yandex.net/" />
 			<test url="http://mobile.photo.yandex.net/" />
 			<test url="http://pythonbp.yandex.net/" />
 
-		<test url="http://api-yaru.yandex.net/" />
 		<test url="http://awaps.yandex.net/" />
 		<test url="http://avatars.yandex.net/" />
 		<test url="http://1.avatars.yandex.net/" />
 		<test url="http://default.avatars.yandex.net/" />
 		<test url="http://avatars-fast.yandex.net/" />
 		<test url="http://ba.yandex.net/" />
-		<test url="http://pix.blogs.yandex.net/" />
-		<test url="http://pix2.blogs.yandex.net/" />
-		<test url="http://browser.yandex.net/" />
-		<test url="http://www.browser.yandex.net/" />
 		<test url="http://captcha.yandex.net/" />
-		<test url="http://a.captcha.yandex.net/" />
-		<test url="http://s.captcha.yandex.net/" />
-		<test url="http://cache-default06f.cdn.yandex.net/" />
-		<test url="http://download.cdn.yandex.net/" />
```

Comment:
Why did you removed this?

Reply:
You may change the test url to something valid, e.g. `http://download.cdn.yandex.net/browser/yandex/ru/beta/Yandex.deb`